tests/testnet.spec.js

```diff
@@ -347,6 +347,35 @@ describe('Testing generic actions', () => {
     expect(result.fee_collected).to.be.a('number')
   })
 
+  it(`getFee for removePublicAddress`, async () => {
+    const result = await fioSdk.genericAction('getFeeForRemovePublicAddress', {
+      fioAddress: newFioAddress
+    })
+
+    expect(result).to.have.all.keys('fee')
+    expect(result.fee).to.be.a('number')
+  })
+
+  it(`Remove public address`, async () => {
+
+    const result = await fioSdk.genericAction('removePublicAddress', {
+      fioAddress: newFioAddress,
+      publicAddresses: [
+        {
+          chain_code: fioChainCode,
+          token_code: fioTokenCode,
+          public_address: 'FIO5kMPnfQw6S3eFqMwTMebYghZAfJjeur8VpvvaaFQnPRSoLxcde',
```

Comment:
I assume we don't need to pass pub address to remove it for chainCode/tokenCode combination

Reply:
the new endpoint on the blockchain verifies the pub address as well as the chain code and the token code to ensure its removing the specified entry, so we need all 3